.github/workflows/main.yml

```diff
@@ -1,5 +1,5 @@
 name: ci
-on: ["push", "pull_request"]
+on: ["push", "pull_request", "workflow_dispatch"]
```

Comment:
okay, don't forgot to remove this before marking this PR as no longer draft, that way i can merge if it looks good

Reply:
Temp change removed: https://github.com/lbryio/lbry-sdk/commit/6a2246c088f8447860be1978c780210a3fcb0828